app/webpacker/components/Results/ResultsFilter.jsx

```diff
@@ -39,34 +49,31 @@ export default function ResultsFilter({ filterState, filterActions }) {
             selectedEvents={[event]}
             onEventSelection={({ eventId }) => setEvent(eventId)}
             hideAllButton
-            hideClearButton
+            hideClearButton={!isRecords}
```

Comment:
```suggestion
            hideClearButton={!isRecords && Boolean(event)}
```
which would match the psych sheet - no clear option is no event is selected.

Reply:
Yup, I implemented this although the boolean logic does need to be flipped here: The `Clear` button should be hidden if `!event`, i.e. if no even is selected.